docs/ref/csrf.txt

```diff
@@ -462,3 +472,30 @@ A number of settings can be used to control Django's CSRF behavior:
 * :setting:`CSRF_FAILURE_VIEW`
 * :setting:`CSRF_HEADER_NAME`
 * :setting:`CSRF_TRUSTED_ORIGINS`
+
+Frequently Asked Questions
+==========================
+
+Is posting an arbitrary CSRF token pair (cookie and POST data) using Firebug, Chrome dev tools, Burp Suite, etc. a vulnerability?
+---------------------------------------------------------------------------------------------------------------------------------
+
+No, this is by design. The comparison between the cookie token and the POST
+token is a server-side check but it's not against any stored server-side value.
+Some security audits flag this as a problem but remember that an attacker
+cannot steal a user's browser's CSRF cookie. "Stealing" or modifying your
+own token isn't a vulnerability.
+
+Is the fact that Django's CSRF protection isn't linked to a session a problem?
+------------------------------------------------------------------------------
+
+No, this is by design. We may link them in the future just so that we don't
+have to argue with auditors about it.
```

Comment:
> As I understand it, the benefit of linking the token to the session is that leaked CSRF tokens are invalidated when the session expires. This won't stop attacks, but it can limit the extend of a successful attack if it goes unnoticed.

Right, though I'd say that this would require an "active" logout from the user cause session times are pretty long usually (not as long as the CSRF cookie though)